recipe/meta.yaml

```diff
@@ -8,7 +8,27 @@ package:
 source:
   - folder: .
     url: https://github.com/DOI-USGS/{{ name }}/archive/{{ version }}.tar.gz
-    sha256: a8ae036caa0b958fa6e28ee8ba7a29c2b6fefeb1413144243d59521f51d1dc12
+    sha256: 2f2b49fc74b6f2eef92839737315be7585b81331d78592bffdf04e31611e8eaf
+
+  # submodules
+  - folder: ale
+    url: https://github.com/DOI-USGS/ale/archive/0.10.0.zip
+    sha256: 2373e0e1ff3715adef60e61db22e6cea1d5b7941f5b1999dd202065d3d84efb2
+  - folder: csm
+    url: https://github.com/USGS-Astrogeology/csm/archive/c78dbba3731432249a75db9fdd7415b292d46715.zip
```

Comment:
This is the wrong URL.

Reply:
Given how old that Org is now, can we fork this over to the new Org? Just in case it just disappears for some reason. I think we can do that without an approval process.